apps/api/src/models/post.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose, { Document, Schema } from 'mongoose';
+
+interface IPost extends Document {
+  title: string;
+  image: string;
+  description: string;
+  category: string;
```

Comment:
`category `also need to be a `Schema.Types.ObjectId` variable to be linked with the Category table

Reply:
category type was updated